source/stylesheets/posts.css.scss

```diff
@@ -1,593 +1,131 @@
-    z-index: 1000;
+body {
+  color: $c-night-rider;
+  font-family: $font-stack-secondary;
+  line-height: 1.6;
+  text-align: left;
 }
 
 
 /* CONTENT & CONTAINERS */
 
-.blogposts__container {
-    align-content: center;
-    background: #f9bf8f;
-    padding-bottom: 20%;
-    padding-left: 15%;
-    padding-right: 15%;
-    width: 100%;
-
-    h2 {
-        column-span: all;
-        text-align: center;
-    }
-
-    ul {
-        list-style-type: none;
-
-        li {
-            padding-top: 15px;
-        }
-    }
-
-    .img {
-        display: block;
-        padding-left: 20%;
-        margin-left: auto;
-        margin-right: auto;
-        width: 100%;
-    }
-}
-
-.blogpost__text--space {
-    padding-top: 5%;
-
-    .column {
-        @media (min-width: 300px) {
-            column-count: 0;
-        }
-
-        @media (min-width: 768px) {
-            column-count: 0;
-        }
-
-        @media (min-width: 900px) {
-            column-count: 2;
-            column-gap: 4em;
-        }
-
-        .typography {
-            column-span: all;
-            padding-top: 3%;
-            text-align: left;
-
-            @media (min-width: 300px) {
-                font-size: 1.15em;
-                text-align: center;
-            }
-
-            @media (min-width: 768px) {
-                font-size: 1.5em;
-                text-align: left;
-            }
-
-            @media (min-width: 900px) {
-                font-size: 1.75em;
-                text-align: left;
-            }
-        }
-
-        h2 {
-            font-size: 1.5em;
-        }
-
-        h3 {
-            @media (min-width: 300px) {
-                font-size: 1em;
-                padding-left: 5%;
-                text-align: left;
-            }
-
-            @media (min-width: 768px) {
-                font-size: 1.25em;
-                padding-left: 0;
-                text-align: left;
-            }
-
-            @media (min-width: 900px) {
-                font-size: 1.25em;
-                text-align: left;
-            }
-        }
-
-
-        h4 {
-            padding-top: 0;
-        }
-
-        p {
-            padding-top: 10px;
-
-            @media (min-width: 768px) {
-                font-size: 1em;
-                text-align: left;
-            }
-        }
-
-        ul {
-            list-style-type: disc;
-
-            li {
-                padding-top: 8px;
-            }
-        }
-    }
-}
-
-.blogposts__space--above {
-    padding-top: 5%;
-}
-
-.blogposts__ul {
-    padding-left: 0;
-
-    li {
-        list-style: disc;
-        font-size: 14px;
-        line-height: 200%;
-        padding-left: 1em;
-    }
-}
-
-.blogposts__ul--none {
-    list-style: none;
-
-    li {
-        list-style: none;
-        padding-left: 1em;
-    }
-}
-
-.blogposts__ul--indent {
-    padding-left: 1.25em;
+main {
+  align-content: center;
+  background: $c-mandys-pink;
+  padding-bottom: 20%;
+  padding-left: 15%;
+  padding-right: 15%;
```

Comment:
Great job refactoring all this scss. One suggestion here is to combine all these paddings into one. The order is clockwise. So for example, you could refactor:
padding-top: 2em;
padding-right: 4em;
padding-bottom: 1em;
padding-left: 5em;

to: 
padding: 2em 4em 1em 5em;

When you refactor this, you can just do 0 for your padding-top (so long as that's for sure what you want -  I would double check to make sure there's not some other padding there that you would want to include!)